packages/node/tests/relay/CallActions.test.ts

```diff
@@ -42,7 +42,8 @@ describe('PlayAction', () => {
 
       const result = await action.stop()
       expect(Connection.mockSend).nthCalledWith(1, stopMessage)
-      expect(result).toEqual(JSON.parse('{"code":"200","message":"message","control_id":"control-id"}'))
+      expect(result).toBeInstanceOf(StopResult)
```

Comment:
@bryanrite please note the change. 
Currently the SDK returns the Relay JSON payload, with this change we return an object instead. 
It is a _soft_ breaking change.

Reply:
Note: to avoid the breaking change i can add `code` and `message` properties to StopResult so - in case a user has inspected the Relay response and is using them - this change will not break his code...

Reply:
Lets do that to be nice. The result was never documented, but if we can avoid breaking anything, lets do it!